Makefile

```
@@ -24,7 +17,7 @@ jupyter_notebook_port := 9003
 jupyter_python_version := 3.8
 jupyter_project_name_python_shared_object := OpenSpaceToolkitMathematicsPy.cpython-38-x86_64-linux-gnu
 
-################################################################################################################################################################
+clang_format_sources_path ?= $(shell find ~+ src/ include/ test/ bindings/python/src/ -name '*.cpp' -o -name '*.cxx' -o -name '*.hpp')
```

```suggestion
clang_format_sources_path ?= $(shell find ~+ src/ include/ test/ bindings/python/src/ -name '*.cpp' -o -name '*.cxx' -o -name '*.hpp' -o -name '*.tpp')
```

Take into account template files as well